yarn-project/aztec.js/src/utils/watcher.ts

```diff
@@ -6,6 +6,9 @@ import { RollupAbi } from '@aztec/l1-artifacts';
 import { type GetContractReturnType, type HttpTransport, type PublicClient, getAddress, getContract } from 'viem';
 import type * as chains from 'viem/chains';
 
+/**
```

Comment:
We need to explicitly state this is only used in tests if this exists within aztec.js, maybe TestWatcher name